tasks/internal_reconcile_report_mismatch/schemas/output.json

```diff
@@ -1,91 +1,95 @@
-              "keyPath",
-              "cumulusArchiveLocation",
-              "orcaEtag",
-              "s3Etag",
-              "orcaLastUpdate",
-              "s3LastUpdate",
-              "orcaSizeinBytes",
-              "s3SizeInBytes",
-              "discrepancyType"
-
-            ],
-          "properties": {
-            "collectionId": {
-              "type": "string",
-              "description": "Cumulus Collection ID value from the ORCA catalog."
-            },
-            "granuleId": {
-                "type": "string",
-                "description": "Cumulus granuleID value from the ORCA catalog."
-              }, 
-            "filename": {
-                "type": "string",
-                "description": "Filename of the object from the ORCA catalog."
-              },
-            "keyPath": {
-                "type": "string",
-                "description": "key path and filename of the object in the ORCA catalog."
-              },
-            "cumulusArchiveLocation": {
-              "type": "string",
-              "description": "Expected S3 bucket the object is located in Cumulus. From the ORCA catalog."
-            },       
-            "orcaEtag": {
-                "type": "string",
-                "description": "etag of the object as reported in the ORCA catalog."
-              },
-            "s3Etag": {
-              "type": "string",
-              "description": "etag of the object as reported in the S3 bucket"
-            },
-            "orcaLastUpdate": {
-              "type": "integer",
-              "description": "The time, in milliseconds since 1 January 1970 UTC, that information was updated in the ORCA catalog."
-            },
-            "s3LastUpdate": {
-              "type": "integer",
-              "description": "The time, in milliseconds since 1 January 1970 UTC, that information was updated in the S3 bucket."
-            },
-            "orcaSizeinBytes": {
-              "type": "integer",
-              "description": "Size in bytes of the object as reported in the ORCA catalog."
+          "s3LastUpdate": {
+            "type": "integer",
+            "description": "The time, in milliseconds since 1 January 1970 UTC, that information was updated in the S3 bucket."
+          },
+          "orcaSizeInBytes": {
+            "type": "integer",
+            "description": "Size in bytes of the object as reported in the ORCA catalog."
           },
-          "s3SizeinBytes": {
+          "s3SizeInBytes": {
             "type": "integer",
             "description": "Size in bytes of the object as reported in the S3 bucket."
-        },
-        "discrepancyType": {
-          "type": "string",
-          "description": "ype of discrepancy found during reconciliation."
-      }
+          },
+          "discrepancyType": {
+            "type": "string",
+            "description": "Type of discrepancy found during reconciliation."
+          },
+          "comment": {
```

Comment:
Open to alternate approaches/expansions to this. Currently can only be 'Error may be due to race condition, and should be checked manually.'